helm-v3/006-repositories.md

```diff
@@ -2,10 +2,22 @@
 
 The following changes will be made to the repository subsystem:
 
+- A new `helm login` command will be added for authenticating against a repository
+- A new `helm push` command will be added for pushing charts to a repository
 - The `helm serve` command will be removed
-- A new `helm push` verb will be added for pushing charts to a repository
+- [ChartMuseum](https://github.com/helm/chartmuseum) will be the Helm subproject
+  dedicated to hosting your own repository
+- [Monocular](https://github.com/helm/monocular) will be the Helm subproject
+  dedicated to distributed repo discovery, and the software powering
+  [Hub](https://github.com/helm/hub)
```

Comment:
Why should we document monocular and hub in the Helm v3 spec? Is there a more appropriate place?

Reply:
@mattfarina fair point. I do think info on subproject responsibilities is important. Where do you suggest this lives?

Reply:
@jdolitsky the readme file for the community repo?